### PR TITLE
Add notebook analysis and move to spacy model

### LINKED ISSUE #66: Create Ability to see Model Changes
In order to reduce prices drastically, we need to use a smaller model. We should set up (in a jupyter notebook probably) the ability to compare how the new model would perform vs the old one based on the runs already submitted by users.

### PR DESCRIPTION
**Pull request by connorjoleary**
Moving to this notebook made some of the responses worse, and made some better. I think that the cost of running a huge model is more than what we loose using this model though.

### CODE REVIEW

File: backend/model/tokenizer.py [hunk 1]
```diff
@@ -1,17 +1,12 @@
...
 from config import config
 import queue as q
 import os
 
-gn_model = KeyedVectors.load_word2vec_format(config['gn_path'], binary=True)
-nlp = spacy.blank('en')
-nlp.vocab.vectors = spacy.vocab.Vectors(data=gn_model.vectors, keys=gn_model.index2word)
-
-#nlp = spacy.load('en_core_web_sm')
+nlp = spacy.load('en_core_web_lg')
```
Comment (deadmau6) [inline backend/model/tokenizer.py:9]: Quick question did you find out that `spacy` is better than Google's Vector stuff?

Comment (connorjoleary) [reply backend/model/tokenizer.py:9]: Not really, but I did to a ton of investigation and found out that if I want to save About $100 a month, then this is the way to do it.

Comment (connorjoleary) [reply backend/model/tokenizer.py:9]: I didn't post the changes, but let me know if you want db access to run them yourself

Comment (deadmau6) [reply backend/model/tokenizer.py:9]: Ahh I see, makes sense

**Review by deadmau6 — APPROVED** [review]

**Pull request merged**